Specs/Scene/Batched3DModel3DTileContentSpec.js

```diff
@@ -146,31 +143,12 @@ defineSuite([
         });
     });
 
-    it('renders with KHR_materials_common extension', function() {
-        // Tests that the batchId attribute and CESIUM_RTC extension are handled correctly
-        return Cesium3DTilesTester.loadTileset(scene, withKHRMaterialsCommonUrl).then(function(tileset) {
-            Cesium3DTilesTester.expectRenderTileset(scene, tileset);
-        });
-    });
-
     it('renders with textures', function() {
         return Cesium3DTilesTester.loadTileset(scene, texturedUrl).then(function(tileset) {
             Cesium3DTilesTester.expectRender(scene, tileset);
         });
     });
 
-    it('renders with compressed textures', function() {
```

Comment:
Removed - no longer testing compressed textures in 3D Tiles 1.0 until a better glTF extension comes around.